ENDPOINTS.md

```diff
@@ -35,10 +35,12 @@ GET /client/about||[**Client.about**](https://github.com/CiscoDevNet/vManage-cli
 POST /device/action/software/package||[**ConfigurationDeviceSoftwareUpdate.upload_software_to_manager**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/device/software_update.py#L23)|[**SoftwarePackageUploadPayload**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/utils/upgrades_helper.py#L68)|None|
-POST /template/policy/definition/control||[**ConfigurationPolicyControlDefinition.create_policy_definition**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/policy/definition/control.py#L25)|[**ControlPolicy**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/definitions/control.py#L299)|[**PolicyDefinitionId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/policy_definition.py#L945)|
-POST /template/policy/definition/control||[**ConfigurationPolicyControlDefinition.create_policy_definition**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/policy/definition/control.py#L25)|[**ControlPolicy**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/definitions/control.py#L299)|[**PolicyDefinitionId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/policy_definition.py#L1006)|
-POST /template/policy/definition/control||[**ConfigurationPolicyControlDefinition.create_policy_definition**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/policy/definition/control.py#L25)|[**ControlPolicy**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/definitions/control.py#L299)|[**PolicyDefinitionId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/policy_definition.py#L1007)|
-POST /template/policy/definition/control||[**ConfigurationPolicyControlDefinition.create_policy_definition**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/policy/definition/control.py#L25)|[**ControlPolicy**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/definitions/control.py#L299)|[**PolicyDefinitionId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/models/policy/policy_definition.py#L1008)|
+POST /disasterrecovery/activate||[**ConfigurationDisasterRecovery.activate**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L94)||[**ActivateDisasterRecoveryTaskId**](https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L24)|
```

Comment:
`https://github.com/CiscoDevNet/vManage-client/blob/main/vmngclient/endpoints/configuration/disaster_recovery.py#L94`

These links are not working